2022/23xxx/CVE-2022-23059.json

```diff
@@ -1,18 +1,105 @@
 {
```

Comment:
'https://github.com/shopizer-ecommerce/shopizer/blob/2.17.0/sm-shop/src/main/java/com/salesmanager/shop/admin/controller/content/StaticContentController.java#L62 ' does not match '^(ftp|http)s?://\\S+$'